src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -40,88 +40,119 @@
-    private $exit_code;
-
-    /**
-     * @param  Phinx\Console\PhinxApplication $app
-     * @param  String                         $default_env  environment fallback, defaults to "development"
-     * @return void
-     */
-    public function __construct(PhinxApplication $app, $default_env = 'development')
-    {
-        $this->app = $app;
-        $this->env = $default_env;
-    }
-
-    /**
-     * Returns the exit code from the last run command.
-     * @return Integer
-     */
-    public function getExitCode()
-    {
-        return $this->exit_code;
-    }
-
-    /**
-     * Returns the output from running the "status" command.
-     * @param  String  $env  environment name (optional)
-     * @return String
-     */
-    public function getStatus($env = null)
-    {
-        $command = ['status', '-e' => $env ?: $this->env];
-        return $this->executeRun($command);
-    }
-
-    /**
-     * Returns the output from running the "migrate" command.
-     * @param  String  $env     environment name (optional)
-     * @param  String  $target  target version (optional)
-     * @return String
-     */
-    public function getMigrate($env = null, $target = null)
-    {
-        $command = ['migrate', '-e' => $env ?: $this->env];
-        if ($target) {
-            $command += ['-t' => $target];
-        }
-        return $this->executeRun($command);
-    }
-
-    /**
-     * Returns the output from running the "rollback" command.
-     * @param  String  $env     environment name (optional)
-     * @param  Mixed   $target  target version, or 0 (zero) fully revert (optional)
-     * @return String
-     */
-    public function getRollback($env = null, $target = null)
-    {
-        $command = ['rollback', '-e' => $env ?: $this->env];
-        if (isset($target)) {
-            // Need to use isset() with rollback, because -t0 is a valid option!
-            // See http://docs.phinx.org/en/latest/commands.html#the-rollback-command
-            $command += ['-t' => $target];
-        }
-        return $this->executeRun($command);
-    }
-
-    protected function executeRun(Array $command)
-    {
-        // Output will be written to a temporary stream, so that it can be
-        // collected after running the command.
-        $stream = fopen('php://temp', 'w+');
-
-        // Execute the command, capturing the output in the temporary stream
-        // and storing the exit code for debugging purposes.
-        $this->exit_code = $this->app->doRun(new ArrayInput($command), new StreamOutput($stream));
-
-        // Get the output of the command and close the stream, which will
-        // destroy the temporary file.
-        $result = stream_get_contents($stream, -1, 0);
-        fclose($stream);
-
-        return $result;
-    }
+	private $app;
+	private $options;
+	private $exit_code;
+
+	/**
+	 * @param PhinxApplication $app
+	 * @param array $options
+	 */
+	public function __construct(PhinxApplication $app, Array $options = array())
```

Comment:
Elsewhere it is uppercase too 
```php
protected function executeRun(Array $command)
	{
		// Output will be written to a temporary stream, so that it can be
		// collected after running the command.
		$stream = fopen('php://temp', 'w+');

		// Execute the command, capturing the output in the temporary stream
		// and storing the exit code for debugging purposes.
		$this->exit_code = $this->app->doRun(new ArrayInput($command), new StreamOutput($stream));

		// Get the output of the command and close the stream, which will
		// destroy the temporary file.
		$result = stream_get_contents($stream, -1, 0);
		fclose($stream);

		return $result;
	}
```
(TextWrapper.php:141)